save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -63,7 +63,9 @@ abstract class Plugin(
         return if (testFiles.isNotEmpty()) {
             rawTestFiles.filter { paths ->
                 // test can be specified by the name of one of it's files
-                paths.any { it.name in testFiles }
+                paths.any { path ->
+                    testFiles.any { it in path.toString() }
```

Comment:
Do we have any tests or at least docs on this mode? Please update them, or add into the main readme, if they are missing.

Reply:
Yes. In main readme in "Executing specific tests" indicates that you need to pass the path to the test file.